packages/stream-object/src/index.ts

```diff
@@ -1,11 +1,24 @@
 import type { StreamTextOptions, StreamTextResult } from '@xsai/stream-text'
 import type { PartialDeep } from 'type-fest'
-import type { Infer, Schema } from 'xsschema'
+import type { Infer, JSONSchema, Schema } from 'xsschema'
 
 import { streamText } from '@xsai/stream-text'
 import { parse } from 'best-effort-json-parser'
 import { toJSONSchema } from 'xsschema'
 
+const wrap = (schema: JSONSchema): JSONSchema => {
+  return {
+    properties: {
+      elements: {
+        items: schema,
+        type: 'array',
+      },
+    },
+    required: ['elements'],
+    type: 'object',
+  }
+}
+
```

Comment:
```suggestion
import { wrap } from '../../generate-object/src/wrap.ts'
```

Reply:
it it appropriate to import a file in another packages...?

Reply:
> it it appropriate to import a file in another packages...?

Yes, this file should be inlined, you can try it

Reply:
The compilation isn't an issue. In terms of logic, generate-object is a separate library, so its path isn't absolute relative to this library.

Reply:
> The compilation isn't an issue. In terms of logic, generate-object is a separate library, so its path isn't absolute relative to this library.

Fair enough.